runtime/local/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "local-runtime"
-version = "4.23.0"
+version = "4.25.0"
```

Comment:
Was this intended? Version goes from 23 to 25.

Reply:
Yes, it seems you missed it in your PR (and me in review) so I bumped it to 25 now.

We always keep the same version for all runtimes and the collator.

Reply:
If I'm not mistaken you said that I shouldn't bump the local runtime version. That's why I omited it in my PR.

Reply:
We shouldn't bump the local runtime's spec version.

The version in the .toml file always gets bumped.

Reply:
Oh, I see. That's an important observation, will add that to the docs. Thank you.